### ISSUE TITLE
[Bug]: 4K video playback is not working (enabled AV1).

### ISSUE THREAD

**Issue by letsconfuse** [FreeTubeApp/FreeTube#4383]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

I just downloaded FreeTube today. I don't know much about it, but I am using uYouTube on my iPhone, so I know that enabling the AV1 format will give 4K playback, but after that, it is not working in this video. I have watched other 4K videos after enabling this AV1.

### Expected Behavior

It should give 4K resolution after enabling the AV1 format.

### Issue Labels

accessibility issue

### FreeTube Version

v0.19.1 Beta

### Operating System Version

Windows 11 23H2

### Installation Method

.zip

### Primary API used

Local API

### Additional Information

![Screenshot 2023-11-25 184134](https://github.com/FreeTubeApp/FreeTube/assets/152010164/5abb34cd-36b3-4f47-a3b4-9fe3702134ef)
![Screenshot 2023-11-25 184427](https://github.com/FreeTubeApp/FreeTube/assets/152010164/fca35045-5c89-4b7f-a249-34020faa030f)
![Screenshot 2023-11-25 190512](https://github.com/FreeTubeApp/FreeTube/assets/152010164/a48a62bd-ef9b-4ddd-aecc-80882c71c5f0)

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
duplicate of https://github.com/FreeTubeApp/FreeTube/issues/2013

**Comment by absidue** [comment]
Not all videos have AV1 formats and it usually takes YouTube a while to create them, the video in your third screenshot is already 10 days old, whereas the one in the first and second screenshot is only a day old.

So why does YouTube have 4k available on that video then? Because YouTube offers their videos in 3 codecs (there are some other combinations but they aren't relevant here):

- h264, in a MP4 container, only up to 1080p, available for all videos
- VP9, in a WebM container, up to highest resolution on the video, available for most videos
- AV1, in a MP4 container, up to highest resolution on the video, only available for some videos and usually only after a while

The video player that FreeTube uses only supports MP4 DASH, which means that it can only use the streams with h264 and AV1 codec. So for videos without AV1 streams, you are limited to 1080p.

**Comment by absidue** [comment]
Closing this issue as this isn't a bug, FreeTube is behaving as expected, because it can't use the AV1 formats, unless YouTube has them, which in this case they don't.
